extensions/wikia/TemplateClassification/scripts/TemplateClassificationLabeling.js

```diff
@@ -52,39 +52,30 @@ define('TemplateClassificationLabeling',
 		})();
 
 		getTitleStrategy = (function getTitleStrategy() {
-			function addTemplate() {
-				return mw.message('template-classification-edit-modal-title-add-template').escaped();
-			}
-
 			function editType() {
 				return mw.message('template-classification-edit-modal-title-edit-type').escaped();
 			}
 
-			function addTypeBeforePublish() {
-				return mw.message('template-classification-edit-modal-select-type-sub-title').escaped();
+			function chooseType() {
+				return mw.message('template-classification-edit-modal-title-select-type').escaped();
 			}
 
 			return {
-				addTemplate: addTemplate,
+				addTemplate: chooseType,
 				editType: editType,
-				addTypeBeforePublish: addTypeBeforePublish
+				addTypeBeforePublish: chooseType
```

Comment:
Left part is for invocation and tells about the place where it's called. The right part tells about the way it's handled within Labeling extension. So it's adding choose type label to modal forced on publish.
Does that make sense now?